assets/src/settings-page/analytics.js

```diff
@@ -54,6 +87,20 @@ function AnalyticsEntry( { entryIndex, onChange, onDelete, type = '', config = '
 		}
 	}, [ config ] );
 
+	const isTextareaFocused = () => {
+		if ( ! textAreaRef?.current ) {
+			return false;
+		}
+		return textAreaRef.current === document.activeElement;
+	};
+
+	const defaultValue = vendorConfigs[ type ]?.sample || '{}';
+
+	/** @type {string} value */
+	const value = ( '' === config.trim() || map( Object.values( vendorConfigs ), ( c ) => c.sample ).includes( config ) ) && ! isTextareaFocused()
```

Comment:
I know we had an issue with `Array.prototype.find` last week, but we're using `Array.prototype.map` all over the place and haven't had any issues:

```suggestion
	const value = ( '' === config.trim() || Object.values( vendorConfigs ).map( ( { sample } ) => sample ).includes( config ) ) && ! isTextareaFocused()
```

Alternatively (slightly quicker and more concise but with `find`):

```suggestion
	const value = ( '' === config.trim() || Object.values( vendorConfigs ).find( ( { sample } ) => sample === config ) ) && ! isTextareaFocused()
```

Reply:
Committed latter suggestion with removal of `map` import in f297e8fad.